conf/client.properties

```diff
@@ -1,31 +1,31 @@
 # Optional. A hostname of the Server. Use localhost by default if not specified.
-#scalar.ledger.client.server_host=localhost
+scalar.ledger.client.server_host=sandbox.scalar-labs.com
```

Comment:
I'm not sure if we should have sandbox settings here.

Reply:
Indeed, I was confused at the time thinking it was just for sandbox. Let's think about what can be done.

Reply:
Actually, let me revert any changes to `client.properties` now so this PR can focus on English changes. @feeblefakie @MDBox Please add some commits if you notice any needed English changes.